tests/integration/lsp_tests.rs

```diff
@@ -6493,136 +6493,6 @@ fn lsp_code_actions_deno_cache_all() {
   client.shutdown();
 }
 
-#[test]
-#[timeout(300_000)]
-fn lsp_code_actions_deno_types_for_npm() {
```

Comment:
Is this behaviour gone? Makes sense we don't need it anymore. If so the implementation revolving around https://github.com/denoland/deno/blob/772fffe686c084cec5e244b29efda793f906bb7b/cli/lsp/tsc.rs#L3728 should be removed

Reply:
Yeah it doesn't trigger anymore because it just uses the `@types` package automatically. I'll remove that bit

Reply:
I guess that snippet was more relevant to https://github.com/denoland/deno/blob/d969c599af7b1c22fb699a7090070e31092ee527/tests/integration/lsp_tests.rs#L8517-L8519. It should be unsupported for auto-imports as well I think.